### PR TITLE
PLT-6995 Do not scroll center channel to bottom when posting in RHS

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Do not scroll center channel to bottom when posting in RHS.

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-6995

### CODE REVIEW

File: webapp/components/post_view/post_list.jsx [hunk 1]
```diff
@@ -117,12 +117,12 @@ export default class PostList extends React.PureComponent {
         this.loadPosts(this.props.channel.id, this.props.focusedPostId);
         GlobalEventEmitter.addListener(EventTypes.POST_LIST_SCROLL_CHANGE, this.handleResize);
 
-        window.addEventListener('resize', this.handleResize);
+        window.addEventListener('resize', () => this.handleResize());
```
Comment (hmhealey) [inline webapp/components/post_view/post_list.jsx]: What's the arrow function needed for? Is `this` not bound by default?

File: webapp/components/post_view/post_list.jsx [hunk 2]
```diff
@@ -210,20 +210,18 @@ export default class PostList extends React.PureComponent {
         }
 
         if (postList && prevPosts && posts && posts[0] && prevPosts[0]) {
-            // A new message was posted, so scroll to bottom if it was from current user
-            // or if user was already scrolled close to bottom
+            // A new message was posted, so scroll to bottom if user
```
Comment (hmhealey) [inline webapp/components/post_view/post_list.jsx]: I think we still want to scroll down if the user was scrolled up, but it was posted in the center channel

File: webapp/components/post_view/post_list.jsx (hunk 1 above)
Comment (jwilander) [inline webapp/components/post_view/post_list.jsx]: So that `event` is not passed through

File: webapp/components/post_view/post_list.jsx (hunk 2 above)
Comment (jwilander) [inline webapp/components/post_view/post_list.jsx]: It still does that see the changes in create_post.jsx

**Pull request merged**